.github/workflows/rust.yml

```
@@ -6,24 +6,47 @@ on:
   schedule:
     - cron: '0 0 1 * *'
 
+defaults:
+  run:
+    working-directory: fiat-rust
+
 jobs:
   test-rust:
 
     runs-on: ubuntu-latest
 
     concurrency:
-      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}-${{ github.job }}
```

Hmm, this doesn't work the way I hoped, and still [cancels the job](https://github.com/mit-plv/fiat-crypto/actions/runs/6076805923/job/16485443157?pr=1648)
Let's try
```suggestion
      group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}-${{ job.name }}
```
Or else just manually disambiguate them?

---

I think it needs ot be at root level